ui/app/components/alphabet-edit.js

```diff
@@ -3,6 +3,82 @@
  * SPDX-License-Identifier: BUSL-1.1
  */
 
-import TransformBase from './transform-edit-base';
+import Component from '@glimmer/component';
+import { tracked } from '@glimmer/tracking';
+import { action } from '@ember/object';
+import { service } from '@ember/service';
+import errorMessage from 'vault/utils/error-message';
 
-export default TransformBase;
+/**
+ * @module AlphabetEdit
+ * `AlphabetEdit` is a component that allows you to create/edit or view an alphabet.
+ *
+ * @example
+ * ```js
+ *   <AlphabetEdit @model={{this.model}} @mode={{this.mode}} />
+ * ```
+ * @param {object} model - This is the transform alphabet model.
+ * @param {string} mode - Is either show, create or edit.
+ */
+
+export default class AlphabetEditComponent extends Component {
+  @service flashMessages;
+  @service router;
+
+  @tracked errorMessage = '';
+
+  get breadcrumbs() {
+    // ideally this is created on the controller in the parent route but this is a generic route and adding breadcrumbs to the controller requires a larger refactor.
+    const { backend } = this.args.model;
+    return [
+      {
+        label: backend,
+        route: 'vault.cluster.secrets.backend.list-root',
+        model: backend,
+        query: { tab: 'alphabet' },
+      },
+      { label: 'Alphabet' },
+    ];
+  }
+
+  transition(route = 'show') {
+    this.displayErrors = '';
```

Comment:
@Monkeychip shouldn't this be `this.errorMessage`? If yes, I can commit the change, just wanted to be sure this wasn't intentional.